### ISSUE TITLE
feat: Support `trailingSlash` option

### ISSUE THREAD

**Issue by techfg** [vernak2539/astro-rehype-relative-markdown-links#11]
Would like to see the [trailingSlash](https://docs.astro.build/en/reference/configuration-reference/#trailingslash) option applied to the generated links. When `trailingSlash='always'`, a `/` would be added, when `never | ignore`, the current behavior would remain since plugin should only be acting on `{.md,.mdx}` files (See related #12).

```markdown
Markdown: [My MD Link](./anotherfile.md)
HTML: <a href="anotherfile/">My MD Link</a>

Markdown: [My MD Link](./anotherfile.md#something)
HTML: <a href="anotherfile/#something">My MD Link</a>

Markdown: [My MD Link](./anotherfile.md?x=something)
HTML: <a href="anotherfile/?x=something">My MD Link</a>
```

Great plugin, solves a major pain point, thank you for your work!

Samples with proposed fixes in PR #21:
- `always`: <https://stackblitz.com/edit/github-pdzrjo-ptwjhv>
- `never`: <https://stackblitz.com/edit/github-pdzrjo-gizrtv>
- `ignore`: <https://stackblitz.com/edit/github-pdzrjo-yp6ehb>

---

**Edit:** Added note about plugin not being able to be used when `trailingSlash='always'`.

**Edit Take 2:** After further testing/findings, edited the OP entirely to focus only on trailing slash behavior for **files that exist** to avoid mixing issues related to directories

**Comment by vernak2539** [comment]
Thanks for the issue submission (and all the others [and your PRs])! I'll be having a look at these this week, hopefully getting them merged and released. Please bear with me as I'm lacking time at the moment 😬

**Comment by techfg** [comment]
@vernak2539 - No worries and happy to contribute!  Ironically, I just pushed a PR for this one :)  Look forward to your review when you have time and open to any suggestions/feedback/questions across all of them. 

Thank you!

**Comment by techfg** [comment]
FYI - Edited OP to Stackblitz samples, one for each `trailingSlash` mode, using the proposed fixes from #21.

**Comment by vernak2539** [comment]
Released in [v0.11.0](https://www.npmjs.com/package/astro-rehype-relative-markdown-links/v/0.11.0). Thanks!
